packages/web/src/scss/components/Toast/index.html

```diff
@@ -224,11 +224,13 @@ <h2 class="docs-Heading">Dynamic Toast Queue</h2>
         <template data-spirit-snippet="item">
           <div class="ToastBar is-hidden" data-spirit-color="inverted" data-spirit-populate-field="item">
             <div class="ToastBar__box">
-              <div class="ToastBar__content">
+              <div class="ToastBar__container">
                 <svg width="20" height="20" aria-hidden="true" data-spirit-populate-field="icon">
                   <use xlink:href="/assets/icons/svg/sprite.svg#info" />
                 </svg>
-                <div class="ToastBar__message" data-spirit-populate-field="message"></div>
+                <div class="ToastBar__content">
+                  <div class="text-truncate-multiline" data-spirit-populate-field="message"></div>
+                </div>
```

Comment:
❗ what is currently not implemented and we need to do, is to add a template option to add a link. 
It means in general:
1. add support in JS plugin
2. add data attribute to link here
3. document it with other "populates".

This might be it.